Tests/Resources/Fixtures/config/full.php

```diff
@@ -1,16 +1,16 @@
 <?php
 
-$container->loadFromExtension('httplug', array(
-    'main_alias' => array(
-        'client' => 'my_client',
+$container->loadFromExtension('httplug', [
+    'main_alias' => [
+        'client'          => 'my_client',
         'message_factory' => 'my_message_factory',
-        'uri_factory' => 'my_uri_factory',
-        'stream_factory' => 'my_stream_factory',
-    ),
-    'classes' => array(
-        'client' => 'Http\Adapter\Guzzle6\Client',
+        'uri_factory'     => 'my_uri_factory',
+        'stream_factory'  => 'my_stream_factory',
+    ],
+    'classes' => [
+        'client'          => 'Http\Adapter\Guzzle6\Client',
```

Comment:
Ah, I see. Well, the reason why I ignored tests by default was that spec tests are snake cased and function names does not contain visibility. I agree it would make sense here, so let's just remove the test paths from exclusions.